parsers/shared/src/main/scala/sigmastate/lang/ContractParser.scala

```diff
@@ -117,8 +117,9 @@ case class ContractSignature(name: String, params: Seq[ContractParam])
  * Includes the parsed docstring preceding the contract as well as the contract signature.
  * @param docs Docstring parsed from the contract.
  * @param signature Signature of the contract.
+ * @param body The code of the contract.
  */
-case class ParsedContractTemplate(docs: ContractDoc, signature: ContractSignature)
+case class ParsedContractTemplate(docs: ContractDoc, signature: ContractSignature, body: String)
```

Comment:
This is better, but still ParsedContractTemplate is only partially parsed.
I think maybe better to parse it completely using SigmaCompiler.parse method.
And then create a new `compileParsed` method in SigmaCompiler, to pass `body` there.

Reply:
Done